presign/base/models.py

```diff
@@ -253,6 +253,7 @@ class ParticipantStates(models.TextChoices):
 
 
 class ParticipantStateActions(models.TextChoices):
+    SUBMIT_APPLICATION = "submit_application"
```

Comment:
Can we document this eventually?

Reply:
This probably also needs a bit of refactoring at some point